manual/source/examples/h5py/plotting/reader_attributes_trail.py

```diff
@@ -1,7 +1,13 @@
-
+import os
```

Comment:
Lately, I've come to prefer *pathlib* to *os* for this pattern:

```py
import pathlib
filename = pathlib.Path(__file__).parent.parent / "simple_example_basic" / "simple_example_basic.nexus.hdf5"

with h5py.File(str(filename), "r") as nx:
...
```

Reply:
Done